src/main/java/uk/gov/justice/digital/delius/transformers/CourtAppearanceTransformer.java

```diff
@@ -29,7 +29,7 @@ public CourtAppearance courtAppearanceOf(uk.gov.justice.digital.delius.jpa.stand
             .crownCourtCalendarNumber(courtAppearance.getCrownCourtCalendarNumber())
             .bailConditions(courtAppearance.getBailConditions())
             .courtNotes(courtAppearance.getCourtNotes())
-            .eventId(courtAppearance.getEventId())
+            .eventId(courtAppearance.getEvent().getEventId())
```

Comment:
Not sure why we return the eventId - I think it is due to New Tech making additional calls to get Offences...

Reply:
Leave it in for now I think, it allows for some wriggle room when traversing the model for other use cases.

Reply:
sure